biome.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
+  "files": {
+    "ignore": [".api", "*.json"],
```

Comment:
I think so too! By the way, where can the file be placed? If it's under a codecov-specific directory, we may want to skip the whole directory to avoid being bothered by similar files in the future. If it can be placed everywhere, this config should be good to go

Reply:
It seems like `.api` stores generated files in ce66298661b2e36cdbd61a2eaa30d1055149b354 from `codecov` [OpenAPI responses](https://api.codecov.io/api/v2/docs/) 📚

AFAIK this `.api` directory could be called something else, but we would just have to update the package path:

https://github.com/slackapi/slack-health-score/blob/b8d57391acabd8189497cf9f9e7cc422cebf37f4/package.json#L37

I'll keep this in mind with future updates, but think this'll be ok to merge as is!

Reply:
oops, it's not "*.api"... I was assuming that it's file extension 🤦 Thanks for sharing the details!